regression-test/data/nereids_tpcds_shape_sf100_p0/shape/query44.out

```diff
@@ -18,24 +18,26 @@ PhysicalResultSink
 --------------------------PhysicalDistribute
```

Comment:
the two phase sort above partition topn is problematic, which will be resolved in later pr.